src/docs/references/marketplace-assets/index.md

```diff
@@ -0,0 +1,111 @@
+---
+title: Marketplace assets
+slug: marketplace-assets
+updated: 2022-05-09
+category: references
+ingress:
+  Reference documentation providing information on Marketplace assets.
+published: true
+---
+
+_Marketplace assets_ are a mechanism defining configuration data and
+content for a marketplace. They are most useful for storing client
```

Comment:
"most useful" sounds a bit off because that's the use-case for them and there aren't really any other use-cases.